torchinductor/sizevars.py

```diff
@@ -11,6 +12,8 @@
 
 from .virtualized import V
 
+log = logging.getLogger(__name__)
```

Comment:
Used logging and removed for this PR?

Reply:
Yeah, I think it is fine to leave this line for the future though.